recipes-kernel/linux/linux-mainline/arm/defconfig

```diff
@@ -2017,28 +2211,29 @@ CONFIG_THERMAL_OF=y
 CONFIG_THERMAL_DEFAULT_GOV_STEP_WISE=y
 # CONFIG_THERMAL_DEFAULT_GOV_FAIR_SHARE is not set
 # CONFIG_THERMAL_DEFAULT_GOV_USER_SPACE is not set
-# CONFIG_THERMAL_DEFAULT_GOV_POWER_ALLOCATOR is not set
 # CONFIG_THERMAL_GOV_FAIR_SHARE is not set
 CONFIG_THERMAL_GOV_STEP_WISE=y
 # CONFIG_THERMAL_GOV_BANG_BANG is not set
 # CONFIG_THERMAL_GOV_USER_SPACE is not set
-# CONFIG_THERMAL_GOV_POWER_ALLOCATOR is not set
 CONFIG_CPU_THERMAL=y
-# CONFIG_CLOCK_THERMAL is not set
+CONFIG_CPU_FREQ_THERMAL=y
```

Comment:
This can lead to throttling the CPU and may have impact on realtime behaviour of sensord.
I suggest to disable CPU_FREQ at all